sql/mysql_db_structure.sql

```diff
@@ -56,9 +56,13 @@ CREATE TABLE `clarification` (
   KEY `probid` (`probid`),
   KEY `cid` (`cid`),
   KEY `cid_2` (`cid`,`answered`,`submittime`),
+  KEY `sender` (`sender`),
```

Comment:
Doctrine assumes we have keys for all foreign keys, so I have added them.

Reply:
We should indeed!